features/org.hibernate.search.eclipse.feature/feature.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.hibernate.search.eclipse.feature"
+      label="%featureName"
+      version="5.2.0.qualifier"
```

Comment:
yes. It is processed together with pom.xml. The artifact should match the feature id, and version number should be the same